en/concept/control_allocation.md

```diff
@@ -0,0 +1,54 @@
+# Control Allocation (Mixing)
+
+
+The PX4 architecture ensures that the airframe layout does not require special case handling in the core controllers.
+
+Mixing means to take force commands (e.g. `turn right`) and translate them to actuator commands which control motors or servos.
+For a plane with one servo per aileron this means to command one of them high and the other low.
+The same applies for multicopters: Pitching forward requires changing the speed of all motors.
+
+Separating the mixer logic from the actual attitude/rate controller greatly improves reusability.
+
+<!-- https://docs.google.com/drawings/d/1Li9YhTLc3yX6mGX0iSOfItHXvaUhevO2DRZwuxPQ1PI/edit -->
+![Mixing Overview](../../assets/diagrams/mixing_overview.png)
+
+
+:::note
+How to setup mixing and outputs is documented [here](../config/actuators.md).
+:::
+
+
+## Actuator Control Pipeline
+
+Overview of the mixing pipeline in terms of modules and uORB topics:
+<!-- https://drive.google.com/file/d/1L2IoxsyB4GAWE-s82R_x42mVXW_IDlHP/view?usp=sharing -->
+<img alt="Pipeline Overview" src="../../assets/concepts/control_allocation_pipeline.png" width="1200" style="text-align: center; margin-left: -230px; margin-right: -230px; max-width: 200%"></img>
```

Comment:
The other image doesn't scale properly because it is fixed width. If we do it this way then the user can click the image to get it full screen.
```suggestion
![Pipeline Overview](../../assets/concepts/control_allocation_pipeline.png)
```